apps/src/code-studio/components/ShareAllowedDialog.jsx

```diff
@@ -282,7 +282,7 @@ class ShareAllowedDialog extends React.Component {
                       type="text"
                       id="sharing-input"
                       onClick={select}
-                      readOnly="true"
+                      readOnly={true}
```

Comment:
nit: for `true` boolean values, you can use a shorthand like this: `<ShareAllowedDialog readOnly />` (whereas an explicitly `false` value would still require `<ShareAllowedDialog readOnly={false} />`)